107/src/test/java/org/ehcache/jsr107/ConfigurationMergerTest.java

```diff
@@ -273,6 +278,53 @@ public void jsr107LoaderInitAlways() {
     assertThat(configHolder.useEhcacheLoaderWriter, is(false));
   }
 
+  @Test
+  public void jsr107DefaultEh107IdentityCopierForImmutableTypes() {
+    XmlConfiguration xmlConfiguration = new XmlConfiguration(getClass().getResource("/ehcache-107-default-copiers.xml"));
+    final DefaultJsr107Service jsr107Service = new DefaultJsr107Service(ServiceLocator.findSingletonAmongst(Jsr107Configuration.class, xmlConfiguration.getServiceCreationConfigurations().toArray()));
+    merger = new ConfigurationMerger(xmlConfiguration, jsr107Service, mock(Eh107CacheLoaderWriterProvider.class));
```

Comment:
This big tests is a single code path for different types. What about testing the other code path?
No XML configuration, XML config without any default copiers, etc ...